website/content/docs/security/acl/auth-methods/index.mdx

```diff
@@ -136,3 +136,30 @@ bearer token for a Consul ACL token by using the login process:
 
 For more details about specific auth methods and how to configure them, click
 on the name of the auth method type in the sidebar.
+
+# Token Locality
+
+By default, Auth Methods create ACL tokens with _local_ scope. ACL tokens with
+local scope are only valid in the local datacenter where they were created.
+Auth Methods created in the primary datacenter can be configured to create ACL
+tokens that have global scope; however, there are limitations depending on the
+type of the Auth Method being used. For all auth methods, setting
+[`-token-locality=global`](/commands/acl/auth-method/create#token-locality) for
+requests to secondary datacenters will fail. 
```

Comment:
Is the assumption that readers have encountered this concept? We seemed to jump from introduction to a specific implementation. It might be a good a idea to start a new paragraph that opens with something like: "You can specify authentication method scope using the `-token-locality` flag. You can set the value to either `global` or `local`. Setting to `global` configures tokens created in the datacenter to provide global access to the resource. Setting to `local` configures tokens created in the local datacenter to provide access to resources within the datacenter. For all authentication methods, setting `-token-locality` to `global` . . . ". 

I may not be stating the information accurately and I know that we'd be repeating a bit of information provided in the CLI section, but I think it would provide additional context to help readers at this point in the docs without having to jump around.    
```suggestion
type of the Auth Method being used. For all auth methods, setting
[`-token-locality=global`](/commands/acl/auth-method/create#token-locality) for
requests to secondary datacenters will fail. 
```

Reply:
yeah, see your point. I like these suggestions. I will update.